explorer/interpreter/type_checker.cpp

```diff
@@ -1029,9 +1029,10 @@ auto TypeChecker::ImplicitlyConvert(std::string_view context,
     destination = destination_constraint;
 
     if (trace_stream_->is_enabled()) {
-      *trace_stream_ << "converting type " << *converted_value
-                     << " to constraint " << *destination_constraint << " for "
-                     << context << " in scope " << impl_scope << "\n";
+      *trace_stream_ << "converting type `" << *converted_value
+                     << "` to constraint `" << *destination_constraint
+                     << "` for " << context << " in scope " << impl_scope
```

Comment:
`ImplScope` printing expects to be on a line of its own, so perhaps this would be better:
```suggestion
                     << "` for " << context << " in scope:\n" << impl_scope
```